server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerRequest.scala

```diff
@@ -5,14 +5,17 @@ import play.utils.UriEncoding
 import sttp.model.{Header, Method, QueryParams, Uri}
 import sttp.tapir.model.{ConnectionInfo, ServerRequest}
 
+import java.net.InetSocketAddress
 import java.nio.charset.StandardCharsets
 import scala.collection.immutable._
 
 private[play] class PlayServerRequest(request: RequestHeader) extends ServerRequest {
   override lazy val method: Method = Method(request.method.toUpperCase)
   override def protocol: String = request.version
   override lazy val uri: Uri = Uri.unsafeParse(request.uri)
-  override lazy val connectionInfo: ConnectionInfo = ConnectionInfo(None, None, None)
+  private val splitHost = request.host.split(":")
+  private val local = if (splitHost.size > 1) Some(new InetSocketAddress(splitHost.head, splitHost.last.toInt)) else None
```

Comment:
isn't that the remote connection info?

Reply:
ah no, that's yet something different. The host/port from the uri isn't really the local socket's address (or the remote's socket's address), so I suppose we shouldn't be using it here